runtime/devnet/src/config/contracts.rs

```diff
@@ -49,13 +41,8 @@ parameter_types! {
 impl pallet_contracts::Config for Runtime {
 	type AddressGenerator = pallet_contracts::DefaultAddressGenerator;
 	type ApiVersion = ();
-	/// The safest default is to allow no calls at all.
-	///
-	/// Runtimes should whitelist dispatchables that are allowed to be called from contracts
-	/// and make sure they are stable. Dispatchables exposed to contracts are not allowed to
-	/// change because that would break already deployed contracts. The `RuntimeCall` structure
-	/// itself is not allowed to change the indices of existing pallets, too.
-	type CallFilter = AllowBalancesCall;
+	// IMPORTANT: only runtime calls through the api are allowed.
+	type CallFilter = Nothing;
```

Comment:
Can we add a test for this please.

It can be as simple as asserting that the callfilter typeid is equal to typeid of Nothing. Any configuration change has a reason, and that reason should be backed by a test. The name of the test can describe the reason, such as 'contracts_prevents_runtime_calls()' with the same comment you have above it.